docs/01-app/04-api-reference/03-file-conventions/layout.mdx

```diff
@@ -86,7 +67,9 @@ export default async function Layout({ params }) {
 
 ### Root Layouts
 
-The `app` directory **must** include a root `app/layout.js`.
+The root layout is defined at the top level of the `app` directory and applies to all routes. This layout is **required** and must contain `html` and `body` tags, allowing you to modify the initial HTML returned from the server.
+
+The `app` router **must** include a root `app/layout.js`.
```

Comment:
The root layout can be under a route group right? So it might not be this specific file path, could be good to clarify